### PR TITLE
update ipvs readme file

### PR DESCRIPTION
**Pull request by Lion-Wei**
Add the way to choose ipvs mode when using kubeadm with a configuration file

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/proxy/ipvs/README.md [hunk 1]
```diff
@@ -47,6 +47,18 @@ kubeadm init --feature-gates=SupportIPVSProxyMode=true
 
 to specify the ipvs mode before deploying the cluster if want to run kube-proxy in ipvs mode.
 
+If you are using kubeadm with a configuration file, you can specify the ipvs mode by adding `featureGates` field.
+Then the configuration file will be look like:
```
Comment (m1093782566) [inline pkg/proxy/ipvs/README.md]: `The configuration file is similar to:` or `The configuration file looks like:` ?

File: pkg/proxy/ipvs/README.md [hunk 2]
```diff
@@ -47,6 +47,18 @@ kubeadm init --feature-gates=SupportIPVSProxyMode=true
 
 to specify the ipvs mode before deploying the cluster if want to run kube-proxy in ipvs mode.
 
+If you are using kubeadm with a configuration file, you can specify the ipvs mode by adding `featureGates` field.
```
Comment (m1093782566) [inline pkg/proxy/ipvs/README.md]: by adding `SupportIPVSProxyMode: true` below the `featureGates` field?

**Comment by m1093782566** [comment]
Please squash the commits, then looks good to me.

**Comment by thockin** [comment]
/approve no-issue

@m1093782566 will LGTM when squashed

**Pull request merged**